cmd/lncli/cmd_invoice.go

```diff
@@ -60,6 +60,14 @@ var addInvoiceCommand = cli.Command{
 				"specified, an expiry of " +
 				"86400 seconds (24 hours) is implied.",
 		},
+		cli.Uint64Flag{
+			Name: "cltv_expiry",
```

Comment:
should we call it `cltv_expiry_delta` just to make it clear that this is in fact a delta and not the absolute expiry?

Reply:
same below

Reply:
fixed